### PR TITLE
Clean up the quiver code

### PR DESCRIPTION
**Pull request by nex3**

### CODE REVIEW

File: lib/fake_async.dart [hunk 1]
```diff
@@ -12,284 +12,235 @@
...
 
-part of quiver.testing.async;
+import 'dart:async';
+import 'dart:collection';
 
-/// A mechanism to make time-dependent units testable.
+import 'package:collection/collection.dart';
+import 'package:quiver/time.dart';
+
+/// The type of a microtask callback.
```
Comment (natebosch) [inline lib/fake_async.dart:21]: [nit] Is this comment adding information?

File: lib/fake_async.dart [hunk 2]
```diff
@@ -12,284 +12,235 @@
...
-  /// Runs all timers until no timers remain (subject to [flushPeriodicTimers]
-  /// option), including those scheduled as a result of running them.
-  ///
-  /// [timeout] lets you set the maximum amount of time the flushing will take.
-  /// Throws a [StateError] if the [timeout] is exceeded. The default timeout
-  /// is 1 hour. [timeout] is relative to the elapsed time.
-  void flushTimers(
-      {Duration timeout: const Duration(hours: 1),
-      bool flushPeriodicTimers: true});
+  /// The queue of microtasks that are scheduled to run when fake time
```
Comment (natebosch) [inline lib/fake_async.dart]: [nit] Avoid repeating type information in the doc comment. "The tasks that are scheduled to run when fake time progresses."

**Review by natebosch — APPROVED** [review]

File: lib/fake_async.dart (hunk 1 above)
Comment (nex3) [reply lib/fake_async.dart:21]: I don't like leaving members undocumented. Even a trivial comment like this implies that there's no extra information that's being left out—it's essentially saying, "this is what it looks like".

File: lib/fake_async.dart (hunk 2 above)
Comment (nex3) [reply lib/fake_async.dart]: Done.

**Pull request merged**